app/src/androidTest/java/com/android/voyageur/ui/trip/TripTest.kt

```diff
@@ -95,5 +100,7 @@ class TripTest {
     assert(trip.activities.isEmpty())
     assert(trip.type == TripType.TOURISM)
     assert(trip.imageUri.isEmpty())
+    assert(trip.photos.isEmpty())
```

Comment:
what is the point of these asserts? IMO testing a data class seems pointless, apart from the equals and hashCode methods.

Reply:
I just updated an already existing test with the new parameters. When I initially wrote the tests I wanted to be sure that we reach 100% line coverage, including that a new trip object is instantiated correctly.

Reply:
Yes, in general this test shouldn't exist. This is a builtin java feature that we don't have any control over so we would reach 100% line coverage on this even by just testing the .equals method.